model/User.ts

```
@@ -0,0 +1,66 @@
+      default: ['user'],
+      required: true,
+    },
+  },
+  {
+    timestamps: true,
+  }
+)
+
+// Delete the model if it exists to prevent the "Cannot overwrite model once compiled" error
+if (mongoose.models.User) {
+  delete mongoose.models.User
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)